pallet-tfgrid/src/lib.rs

```diff
@@ -30,6 +30,8 @@ pub type BalanceOf<T> =
 pub trait Config: system::Config + timestamp::Config {
     type Event: From<Event<Self>> + Into<<Self as frame_system::Config>::Event>;
     type Currency: Currency<Self::AccountId>;
+    /// Origin for sudo extrinsics
+    type CouncilOrigin: EnsureOrigin<Self::Origin>;
```

Comment:
I would try to find a better name here, besides the fact that "Council" is an internal working term to grasp the DAO better, from the pallet's perspective it is an Origin that protects the priviliged functionality and is configured externally ( can be a combination of origin rules).